src/listing.rs

```diff
@@ -95,6 +95,9 @@ pub struct Entry {
 
     /// Last modification date
     pub last_modification_date: Option<SystemTime>,
+
+    /// Path of symlink pointed to
+    pub symlink_dest: Option<String>,
```

Comment:
I was wondering whether `std::fs::read_link` will fail or not because the description of the [doc](https://doc.rust-lang.org/std/fs/fn.read_link.html) says 
> This function will return an error in the following situations, but is not limited to just these cases
> - path is not a symbolic link.
> - path does not exist.

If this won't fail, I will be glad to remove `is_symlink` and use only `symlink_dest` only.

Thanks for reviewing!

Reply:
I don't think there is a point of showing the `symlink-symbol` without the symlink destination. In case of failure, you  can choose to silently fail and never show them both, or return `500` response.

Reply:
Even if we want to indicate failure, I think the idiomatic way is 
```rust
pub symlink_info: Option<(Option<String>, )>
```
Or we may store empty string!

Reply:
Empty string sounds good enough to me.